tests/test_pipeline_output.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import numpy as np
+import os
+import allensdk.core.json_utilities as ju
+
+
+import sys
+sys.path.append("/local1/pyprojects/allensdk.ipfx/bin") # there is got to be a better way
+from run_pipeline import run_pipeline
+
+
+def check_json_files_are_equal(benchmark_json,test_json):
+
+    benchmark_pipeline_output = ju.read(benchmark_json)
+    test_pipeline_output = ju.read(test_json)
+
+    return benchmark_pipeline_output == test_pipeline_output
+
+
+def test_pipeline_output():
+    """
+    Validates pipeline output for a list of specimens.
+    Runs pipeline, saves data to disk and compares pipeline output to that from the benchmark runs.
+    If pipeline output has changed (was not validated), raises assertion error and lists specimens failing validation.
+
+
+    """
+    specimens = np.array([500844783, 509604672, 513876493])
+
+    validation = []
+
+    for specimen in specimens:
+        print("testing specimen %s" % specimen)
+        benchmark_dir = "/allen/aibs/informatics/module_test_data/ipfx/benchmark_pipeline_output/%s" % specimen
```

Comment:
Don't hardcode these paths into your script.  These need to be configurable.  Do this by passing in a configuration file in an environment variable and putting this at the top of this file:

```python
from pkg_resources import resource_filename
import resource_package
if 'TEST_SPECIMEN_FILE' in os.environ:
    TEST_SPECIMEN_FILE = os.environ['TEST_SPECIMEN_FILE']
else:
    TEST_SPECIMEN_FILE = resource_filename(__name__, 'test_specimens.txt')

with open(TEST_SPECIMEN_FILE,'rb') as f:
    TEST_SPECIMENS = list(f)
```

Reply:
Also, your `TEST_SPECIMENS` should not be a list of IDs that you append to a directory -- it should be fully-qualified paths to `pipeline_input.json`.